<launch>

    <include file="$(find rosbot_description)/launch/rosbot_rviz.launch" />
    <include file="$(find rosbot_navigation)/launch/rosbot_teleop.launch" />

    <node name = "test" pkg="visual_slam" type="test_node" output="screen">
        <remap from="/camera/image_raw" to="/camera/rgb/image_raw"/>
        <remap from="/camera_info" to="/camera/rgb/camera_info"/>
    </node>




</launch>